tests/Utilities/testframework.f90

```diff
@@ -0,0 +1,143 @@
+!> \file
```

Comment:
That's fine if you want those macros to be upper case to match gtest. Other internal variables and the rest of the code should be according to the code style.